EventKit/eventkit-cloud
# Refactor account pages

# Description of Improvement

Refactors the account page components to be functional components. Refactors account component tests to use react testing library.

## How to test

Ensure you have local environment setup with licenses and providers.
Verify account page still appears on first login
Verify licenses can be accepted and checkboxes work as expected
 - accepted and saved licenses cannot be unchecked
 - all checkbox respects above case
After accepting and navigating to dashboard, navigate back to account page and verify checkboxes still appear checked
Verify tests pass

## Quality Assurance

The following items have been updated:


## Screenshots

:smile:
